columbia-html.html

```diff
@@ -268,60 +305,62 @@ <h3 id="converting-plain-text-html">4. Converting Plain Text HTML</h3>
-      versus &lt;span style ="font-size:120"%&gt;...&lt;/span&gt;).
-    </blockquote>
-  </p>
-  <P>
-    <b>This sentence is bold.</b> <i> This sentence is in italics.</i> <b><i>This sentence
-        is in bold italics. </i></b> This sentence is in typewriter font. This sentence has <u>underlined
-      words</u> and <u><b>underlined bold words.</b></u>. This sentence has colored words. This sentence has big
-    words. This
-    one has very big words. This one has very small words.
-  </P>
-  <blockquote>
-    This is a "blockquote", which is like a regular paragraph, but it has bigger margins. Begin a blockquote
-    with &lt;blockquote&gt; and end it with &lt;/blockquote&gt;. Environments such as blockquotes, lists, etc,
-    that have a
-    beginning and an end always use paired commands like &lt;blah&gt;...&lt;/blah&gt;.
+  <div class="chapter-6">
+    <h3 style="color: green" id="effects">5. Effects</h3>
+    <p>
+      The rest of this document shows some of what you can do with simple HTML commands,
+      but I don't explain how to do it. To see that, just tell your browser to
+      <b>View Source</b> and compare the HTML in the source window with the result
+      in the original window.
+    </p>
+    <p>
+      <blockquote>
+        <i>Note:</i> In this and the following sections, I use some "deprecated" features from earlier
+        HTML versions because they are easier for beginners to learn (for example &lt;big&gt;...&lt;/big&gt;
+        versus &lt;span style ="font-size:120"%&gt;...&lt;/span&gt;).
+      </blockquote>
+    </p>
+    <P>
+      <b>This sentence is bold.</b> <i> This sentence is in italics.</i> <b><i>This sentence
+          is in bold italics. </i></b> This sentence is in typewriter font. This sentence has <u>underlined
+        words</u> and <u><b>underlined bold words.</b></u>. This sentence has colored words. This sentence has big
+      words. This
+      one has very big words. This one has very small words.
+    </P>
     <blockquote>
-      This is a blockquote inside another blockquote, which shows how HTML environments can be "nested".
+      This is a "blockquote", which is like a regular paragraph, but it has bigger margins. Begin a blockquote
+      with &lt;blockquote&gt; and end it with &lt;/blockquote&gt;. Environments such as blockquotes, lists, etc,
+      that have a
+      beginning and an end always use paired commands like &lt;blah&gt;...&lt;/blah&gt;.
+      <blockquote>
+        This is a blockquote inside another blockquote, which shows how HTML environments can be "nested".
+      </blockquote>
+      Here we are back in the first blockquote again.
     </blockquote>
-    Here we are back in the first blockquote again.
-  </blockquote>
-  And here we are back outside of the first blockquote.
+    And here we are back outside of the first blockquote.
+  </div>
   <hr>
-  <h3 id="lists">6. Lists</h3>
-  <p>
+  <h3 style="color: grey; background-color: white" id="lists">6. Lists</h3>
```

Comment:
akkor szerintem legyen minden `h3`-nak ilyen stilusa, ugy egysegesebb

Reply:
Azért lett így készítve, hogy beletegyek minden féle megoldást a kódba, hogy lássátok, hogy több féle eljárást is elsajátítottam.